pkg/webhook/inject/inject.go

```diff
@@ -520,19 +507,19 @@ func mergeVolumeMounts(volumeMounts []corev1.VolumeMount, containers []corev1.Co
 
 func updateAnnotations(target map[string]string, added map[string]string) (patch []patchOperation) {
 	for key, value := range added {
-		keyEscaped := strings.Replace(key, "/", "~1", -1)
-
 		if target == nil || target[key] == "" {
 			target = map[string]string{}
 			patch = append(patch, patchOperation{
-				Op:    "add",
-				Path:  "/metadata/annotations/" + keyEscaped,
```

Comment:
why not need keyEscaped

Reply:
use `k-v` type to set annotations, so `keyEscaped` is unnecessary